app/controllers/api/maintenance_stats_controller.rb

```diff
@@ -37,13 +41,17 @@ def begin_watching_repositories
 
     # compact repos array to remove any invalid/not found repositories
     repos.compact.each do |repo|
+      # mark it as now interesting; someday we may make a dedicated endpoint for this,
+      # which would allow us to mark repos that don't support maint stats perhaps.
+      repo.update(interesting: true)
```

Comment:
this'll do a lot of work re: callbacks:

* `before_save :normalize_license_and_language`: re-parses the language col
* `after_commit :save_projects, on: :update`: calls `forced_save` on all repo's projects
  * `Project#forced_save`: 
    * updates Project#updated_at
    * `Project.after_commit :update_repository_async, on: :create`
      * calls `Project#update_repository`
    * `Project.after_commit :update_source_rank_async, on: %i[create update]`
    * `Project.after_commit :send_project_updated, on: %i[create update]`
    * `Project.before_save  :update_details`

wasn't sure if that was intentional, so just wanted to suggest an `update_column(interesting: true)` if we don't need the callbacks.

Reply:
agreed, good idea.

Note that I also killed that project.forced_save though in this PR (for unrelated reasons, I actually killed it so regenerating the metrics wouldn't do the Project callbacks) https://github.com/librariesio/libraries.io/pull/3515/commits/808a04a22723509d5c32c1728265ef30b6348fd6